src/formatting_stack/linters/eastwood.clj

```diff
@@ -32,10 +32,11 @@
     (->> @reports
          :warnings
          (map :warn-data)
-         (remove (fn [{{{[[_fn* [_arglist [_assert v]]]] :form} :ast} :wrong-pre-post}]
-                   (= "*" ;; False positives for dynamic vars https://git.io/fhQTx
-                      (-> v str first)
-                      (-> v str last))))
+         (remove (fn [{{{[_fn* [_arglist [_assert v]]] :form} :ast} :wrong-pre-post}]
```

Comment:
Maybe we can make this fn a defn and unit-test it?

...sucks a bit b/c this is a temp patch for https://github.com/jonase/eastwood/pull/333, but it's desirable to rest assured that our code can handle all cases:

* fake pre-post (`(defn x [y] {:pre [*dyn*]})`, where `{:pre [*dyn*]}` is the return value, not a precondition)
* un-namespaced pre
* namespaced pre